README.md

```diff
@@ -66,6 +66,37 @@ of your `pom.xml` and the `dagger-compiler` artifact as either an `optional` or
 </dependencies>
 ```
 
+or as an `annotationProcessorPaths` value of the `maven-compiler-plugin`
```

Comment:
* requires `maven-compiler-plugin` version 3.5+
* things like `mvn dependency:go-offline`, `mvn versions:display-dependency-updates` won't see them (of course, if you use a property for the Dagger version, `mvn versions:display-property-updates` would see it from the `com.google.dagger:dagger` _standard_ dependency)
* doesn't respect `dependencyManagement` (for managing versions), forcing you to use a property. In other words, you have `<version>` scattered all over your POMs, rather than centralized in a `dependencyManagement`; and it prevents you from using BOMs (`<scope>import</scope>`) to manage versions of your dependencies across projects of your organization (forces you to use a `<parent>` defining a property for the Dagger version).